modules/virt-vm-storage-volume-types.adoc

```diff
@@ -47,13 +47,26 @@ the container image registry. The image is pulled from the registry and
 embedded in a volume when the virtual machine is created. A
 *containerDisk* volume is ephemeral. It is discarded when
 the virtual machine is stopped, restarted, or deleted.
+This means that they are useful *only* for read-only filesystems such
```

Comment:
Let's make this stand out more, i.e. in a box

Reply:
@bgaydosrh Is it ok to add an info box inside a table? Would this even compile?
Can you otherwise review my suggestions?

Reply:
Hi @dankenigsberg - Usually we steer clear of putting admonitions in table cells because they can trigger accessibility and translation problems. Since you have an example to add I'd recommend relocating this info to a task module... where we *can* add the admonition to highlight the important pieces. Let me research this and I'll get back to you shortly. We can also update this module (virt-vm-storage-volume-types) without the admonition if needed but this is a less-trafficked reference module. I'll contact our DPM and see if we can get these requirements into a Jira story. Is there an existing epic we can use to track?  Thanks, Bob

Reply:
Thanks, @bgaydosrh. This PR intends to fix https://bugzilla.redhat.com/show_bug.cgi?id=1875725
If we cannot add a formal admonition, I believe @fabiand would be fine with plain (or blodface) textual warning.

Reply:
@aburden has this bug and should be in communication with you @dankenigsberg. Thanks for alerting us.